.github/actions/merge-artifacts/action.yml

```diff
@@ -0,0 +1,76 @@
+name: "Merge Artifacts"
+description: "Merge Artifacts"
+inputs:
+  artifact_name:
+    type: string
+    required: true
+  artifact_files:
+    type: string
+    required: true
+  artifact_pattern:
+    type: string
+    required: true
+  condition:
+    type: boolean
+    required: true
+  github_token:
+    required: true
```

Comment:
you shoulld absoultely install github vscode extension
![image](https://github.com/centreon/centreon/assets/11978823/39681665-453b-47f4-b8e9-405026823136)